src/ontology/uberon-edit.obo

```diff
@@ -102919,7 +102919,7 @@ id: UBERON:0005455
 name: interventricular groove
 def: "Either of the anterior and posterior grooves on the surface of the heart that lie over the interventricular septum and join at the apex." [http://www.merriam-webster.com/medical/interventricular%20sulcus]
 subset: pheno_slim
-synonym: "interventricular sulcus" RELATED [FMA:7223]
+synonym: "interventricular sulcus" EXACT [FMA:7223]
```

Comment:
References to support this being a synonym:
https://www.merriam-webster.com/medical/interventricular%20sulcus
https://medical-dictionary.thefreedictionary.com/interventricular+groove

Reply:
```suggestion
synonym: "interventricular sulcus" EXACT [FMA:7223, https://www.merriam-webster.com/medical/interventricular%20sulcus]
```